### DISCUSSION TITLE
Update for aniyomi

### DISCUSSION THREAD

**Discussion by Raven0770** [aniyomiorg/aniyomi#701] [General]
When will we get best update for aniyomi

**Comment by sinnghabhay80** [comment]
Hate to break it you but there is never a thing as best or perfect.🥺🥺🥲🥲🥲

**Comment by Raven0770** [comment]
May not be best but better

**Comment by sinnghabhay80** [comment]
Ha🙂, Well if you are using the mpv version then we don't actually know what improvement s will be there but if you are using normal version you can look forward to the stable release of mpv version with revamped player and more things or you can go and download the mpv-beta version now also.
